### PR TITLE
Disable SSE optimization flags - OpenSSL dislikes it

### PR DESCRIPTION
**Pull request by ksubrama**

### CODE REVIEW

File: lib/omnibus/software.rb [hunk 1]
```diff
@@ -571,11 +571,17 @@ def with_standard_compiler_flags(env = {}, opts = {})
           freebsd_flags
         when "windows"
           arch_flag = windows_arch_i386? ? "-m32" : "-m64"
-          opt_flag = windows_arch_i386? ? "-march=i686" : "-march=x86-64"
+          #opt_flag = windows_arch_i386? ? "-march=i686" : "-march=x86-64"
```
Comment (jaym) [inline lib/omnibus/software.rb]: I think having your comments is fine, but you should just delete the commented out code. You can always come back and get it

File: lib/omnibus/software.rb [hunk 2]
```diff
@@ -571,11 +571,17 @@ def with_standard_compiler_flags(env = {}, opts = {})
...
           {
             "LDFLAGS" => "-L#{install_dir}/embedded/lib #{arch_flag}",
-            # If we're happy with these flags, enable SSE for other platforms running x86 too.
-            "CFLAGS" => "-I#{install_dir}/embedded/include #{arch_flag} -O3 -mfpmath=sse -msse2 #{opt_flag}"
+            # If we're happy with these flags, enable SSE for other platforms
```
Comment (jaym) [inline lib/omnibus/software.rb]: Is this still true?

**Comment by jaym** [comment]
I think this puts us closer to what all the knapbuild things did, so I'm :+1:

File: lib/omnibus/software.rb (hunk 1 above)
Comment (btm) [inline lib/omnibus/software.rb]: is there a comment about why the -march is removed too?

File: lib/omnibus/software.rb (hunk 2 above)
Comment (ksubrama) [inline lib/omnibus/software.rb]: Well - that's there for Lamont :)  He wanted to do it on solaris and other slow platforms.

File: lib/omnibus/software.rb (hunk 1 above)
Comment (ksubrama) [inline lib/omnibus/software.rb]: It was overriding openssl's explicit -march i486 flag.  I'll add a comment to that effect too.

@jaym I was leaving the code in for context as to what cannot be done without investigation.  It's easy for someone to accidentally enable a flag without checking history to see if that flag was disabled.

**Comment by lamont-granquist** [comment]
can we enable SSE everywhere but disable it on openssl?

**Comment by danielsdeleo** [comment]
Yeah, seems likely that OpenSSL is a special case.

**Comment by ksubrama** [comment]
Withdrawn given the above PR in omnibus-software.

**Pull request closed without merging**